aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/container/TabsHandler.java

```diff
@@ -117,9 +112,9 @@ private void appendAttributes(Element tabElement, Tab tab){
      * @param tab {@link Tab} annotation to analyze
      * @return True or false
      */
-    private static boolean isFieldForTab(Field field, Tab tab) {
+    private static boolean isFieldForTab(Field field, Tab tab, boolean firstTab) {
```

Comment:
name it `isFirstTab` (never mind the argument name is the same as the variable above)